palantirnet/ama-style-guide-joe
# Ewlj 69 header navigation prototype

## Ticket(s)

**Github Issue**
N/A

**Jira Ticket**

- [EWLJ-69: Header & Navigation Prototype](https://issues.ama-assn.org/browse/EWLJ-69)

## Description
Create Header & Navigation components. This includes:
1. Utility Nav Molecule
1. Primary Nav Molecule
1. Site Search molecule
1. Drawer Trigger Atom
1. Wordmark Molecule
1. Header Organism
1. Icons atom

## To Test
- Spin up patternlab
- Review the molecules listed above.
- Review the header organism as a whole.
- Review the sticky functionality (see article page for sticky functionality)
- Review sass/markup/js

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A
